source/OutfitterPanel.cpp

```diff
@@ -46,6 +46,22 @@ namespace {
 	{
 		return to_string(tons) + (tons == 1 ? " ton" : " tons");
 	}
+	
+	// Retrieve the relevant planetary storage for the given outfit
+	CargoHold *GetContainingStorage(const Planet *planet, map<const Planet *, CargoHold *> systemStorage, const Outfit *outfit)
```

Comment:
Where we know the pointers can't be null, we shouldn't add new code that uses pointers over references. So this signature should use `const Planet &planet` and `const Outfit &outfit`.

Also, this currently copies `systemStorage`, but the code below would work just fine on a const reference to it, once you replace the use of `operator[]` on the map with the iterator-based access.